website/docs/reference/sinks/humio.md.erb

```diff
@@ -0,0 +1,21 @@
+<%- component = metadata.sinks.humio -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+## Options
+
+<%= options(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
+<%= component_sections(component) %>
```

Comment:
Can we add a section on obtaining your Humio key?

Reply:
Yeah, I noted this in the PR comment, I'm not actually sure what they would prefer here. I can try to get something simple for now but I assume they would have a better idea.

Reply:
Do you mean how to get the ingest token? (https://docs.humio.com/ingesting-data/ingest-tokens/)

For cloud, you get a login and a repository then go to the Setting page.
Basically the same for on-prem except that we're not the agent adding the login to the cluster or repository.

Reply:
Yeah, I included that link here https://github.com/timberio/vector/pull/1716/files#diff-9cd3df18869e0d811f4a5ddd2f23a33cR21-R25

If you have any suggestions for improvements here from your end I'd love to include them :)